### PR TITLE
Feature qubole hook support headers

### LINKED ISSUE #15598: Qubole Hook Does Not Support 'include_headers'
**Description**

Qubole Hook and Operator do not support `include_header` param for getting results with headers
Add Support for `include_header` get_results(... arguments=[True])
**Use case / motivation**

It's very hard to work with CSV results from db without headers. 
This is super important when using Qubole's databases.

**Are you willing to submit a PR?**

Not sure yet, I can give it a try

**Related Issues**

### PR DESCRIPTION
**Pull request by levyitay**
closes: #15598
related: #15598

---
Added support to include column headers in the result from Qubole.

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/providers/qubole/operators/qubole.py [hunk 1]
```diff
@@ -250,9 +250,9 @@ def on_kill(self, ti=None) -> None:
         else:
             self.get_hook().kill(ti)
 
-    def get_results(self, ti=None, fp=None, inline: bool = True, delim=None, fetch: bool = True) -> str:
+    def get_results(self, ti=None, fp=None, inline: bool = True, delim=None, fetch: bool = True, include_headers: bool = False) -> str:
         """get_results from Qubole"""
-        return self.get_hook().get_results(ti, fp, inline, delim, fetch)
+        return self.get_hook().get_results(ti, fp, inline, delim, fetch,include_headers)
```
Comment (uranusjr) [inline airflow/providers/qubole/operators/qubole.py]:
```suggestion
        return self.get_hook().get_results(ti, fp, inline, delim, fetch, include_headers)
```

Before pylint wastes a CI run to catch it…

Comment (levyitay) [reply airflow/providers/qubole/operators/qubole.py]: Thanks, not sure why I missed it. 
Fixed along with some other warnings

**Review by eladkal — APPROVED** [review]
LGTM

@levyitay  please fix the static checks

**Comment by levyitay** [comment]
@eladkal Done

**Pull request merged**